src/item_factory.cpp

```diff
@@ -1664,7 +1664,7 @@ void Item_factory::load_basic_info( JsonObject &jo, itype &def, const std::strin
     }
 
     if( jo.has_string( "description" ) ) {
-        def.description = _( jo.get_string( "description" ).c_str() );
+        def.description = jo.get_string( "description" ).c_str();
```

Comment:
The call to `c_str` here is pointless. You have a string, get a `const char*` from it only to convert it back into a string. Remove the `.c_str()`.